### PR TITLE
Backfill oldest changelog items

### PR DESCRIPTION
**Pull request by irvinebroque**
Backfills past changes to the Stream changelog.

### CODE REVIEW

**Comment by Princeobinna** [comment]
Approve assets

File: content/stream/changelog.md [hunk 1]
```diff
@@ -65,15 +65,136 @@ The [Stream Player](/stream/viewing-videos/using-the-stream-player/) now shows t
...
+
+### 24/7 Live streaming support
+
+You can now use Cloudflare Stream for 24/7 live streaming.
+
+## 2021-11-17
+
+### Persistent Live Stream IDs
+
+You can now start and stop live broadcasts without having to provide a new video ID to the Stream Player (or your own player) each time the stream starts and stops. [Read the docs](https://developers.cloudflare.com/stream/stream-live/watch-live-stream/#view-by-live-input-id).
```
Comment (irvinebroque) [inline content/stream/changelog.md]: update link

File: content/stream/changelog.md [hunk 2]
```diff
@@ -65,15 +65,136 @@ The [Stream Player](/stream/viewing-videos/using-the-stream-player/) now shows t
...
+
+For more, see [this post](https://community.cloudflare.com/t/announcing-the-preview-build-for-stream-player-2-0/243095) on the Cloudflare Community Forum.
+
+## 2021-03-04
+
+### Faster video encoding
+
+Videos uploaded to Cloudflare Stream are now available to view 5x sooner, reducing the time your users wait between uploading and viewing videos.
+
+## 2021-05-03
```
Comment (irvinebroque) [inline content/stream/changelog.md]: Wrong order

**Review by bkrebsbach — APPROVED** [review]

**Review by third774 — APPROVED** [review]

File: content/stream/changelog.md [hunk 3]
```diff
@@ -65,15 +65,136 @@ The [Stream Player](/stream/viewing-videos/using-the-stream-player/) now shows t
...
+### Stream Player 2.0 (preview)
+
+A brand new version of the Stream Player is now available for preview. New features include:
+
+- Unified controls across desktop and mobile devices
+- Keyboard shortcuts
+- Intelligent mouse cursor interactions with player controls
+- Phased out support for Internet Explorer 11
+
+For more, see [this post](https://community.cloudflare.com/t/announcing-the-preview-build-for-stream-player-2-0/243095) on the Cloudflare Community Forum.
```
Comment (dcpena) [inline content/stream/changelog.md]: Let's swap out "see" for something like "refer to" to be more inclusive of visually impaired users.

Comment (irvinebroque) [reply content/stream/changelog.md]: fixed: https://github.com/cloudflare/cloudflare-docs/pull/5174/files#diff-8439a0a77cd41beb0e0e683b7ca7d98dacdf8152eaee6ab0effe1f1bf311d122R147

**Pull request merged**